### PR TITLE
Create Security Policy for Kube API Load Balancer

### PR DESCRIPTION
**Pull request by mnitchev**
Finishes giantswarm/roadmap#1196

### CODE REVIEW

**Comment by mnitchev** [comment]
giantswarm/roadmap#1293 Is a follow up on this, hence the default rules for the VPN and NAT gateway are missing in this PR

File: controllers/gcpcluster.go [hunk 1]
```diff
@@ -137,10 +162,37 @@ func (r *GCPClusterReconciler) reconcileDelete(ctx context.Context, gcpCluster *
 	return ctrl.Result{}, nil
 }
 
-func (r *GCPClusterReconciler) createBastionFirewallRule(ctx context.Context, logger logr.Logger, gcpCluster *capg.GCPCluster) error {
+func (r *GCPClusterReconciler) applyAPISecurityPolicy(ctx context.Context, logger logr.Logger, gcpCluster *capg.GCPCluster) error {
+	policyName := getAPISecurityPolicyName(gcpCluster.Name)
+	sourceIPRanges, err := getIPRangesFromAnnotation(logger, gcpCluster, AnnotationAPIAllowListSubnets)
```
Comment (calvix) [inline controllers/gcpcluster.go:167]: IMHO the annotation should only contain customer IPs, our IP of VPN and MC outgoing Ips should be added by some other way so we do not confuse customers and also we don't allow customer to remove our IPs from the list

**Review by calvix — APPROVED** [review]
was too slow to read , LGTM

**Comment by fiunchinho** [comment]
We are missing a mention to the changes in the changelog.

**Pull request merged**